src/components/AddElement.tsx

```diff
@@ -13,10 +13,12 @@ const PressableContainer = styled(PressableOpacity)`
 const AddCustomDisciplineText = styled(SubheadingPrimary)`
   text-transform: uppercase;
   padding-left: ${props => props.theme.spacings.xs};
+  margin-right: ${props => props.theme.spacings.xs};
 `
 
 const Explanation = styled(ContentSecondary)`
-  padding: 0 ${props => props.theme.spacings.xxl} 0 11%;
+  padding: 0 ${props => props.theme.spacings.xxl} 0 ${props => props.theme.spacings.xs};
+  font-family: ${props => props.theme.fonts.contentFontRegular};
```

Comment:
Sorry about the confusion here. It is not needed to set the font-family here, as it is inherited from ContentSecondary. Please remove font-family here.